package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dotnet-build",
-  "version": "8.0.1",
+  "version": "8.0.2",
```

Comment:
@HBTeun , whenever a dependency that you expost updates a minor version you should also do so.

Please increase to 8.1.0 :)
I'll release 8.1.0 now.